.github/workflows/ci.yaml

```diff
@@ -76,6 +84,10 @@ jobs:
     strategy:
       matrix:
         node-version: [16.x, 18.x]
+    permissions:
+      statuses: write
+      pull-requests: write
+      packages: read
```

Comment:
these should all be unnecessary? I don't think we're adding anything to the status checks or commenting on the PR in this job? Also the base permissions of the org should have `packages: read` by default